templates/cmd/controller/main.go.tpl

```diff
@@ -36,6 +36,13 @@ func main() {
 	flag.Parse()
 	ackCfg.SetupLogger()
 
+	if err := ackCfg.Validate(); err != nil {
+	    setupLog.Error(err, "Unable to create controller manager",
+	    "aws.service", awsServiceAlias,
+    	)
+    	os.Exit(1)
+    }
```

Comment:
nit: with `golint` here it will look like:

```go
	if err != nil {
		setupLog.Error(
			err, "unable to create controller manager",
			"aws.service", awsServiceAlias,
		)
		os.Exit(1)
	}
```